Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/FlyoutSamplePage.xaml

```diff
@@ -6,7 +6,8 @@
 	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
 	  xmlns:smtx="using:ShowMeTheXAML"
 	  xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-	  mc:Ignorable="d">
+	  xmlns:wasm="http://uno.ui/wasm"
```

Comment:
Of note, there are many platforms: win, macos, wasm, android, ios, skia.gtk (linux)
So it is not just win+wasm, when you take out mobile platforms.

Reply:
Good catch. I'll make the necessary changes

Reply:
@Xiaoy312 Latest commit should solve this 👍